tests/test_codelists.py

```diff
@@ -260,3 +260,87 @@ def test_codelists_check_with_upstream_changes_in_CI(
     os.chdir(codelists_path)
     # check doesn't fail in CI if there are upstream errors only
     assert codelists.check()
+
+
+def test_codelists_add(codelists_path, requests_mock):
+    codelists_path /= "codelists"
+    codelists_file = codelists_path / "codelists.txt"
+    prior_codelists = codelists_file.read_text()
+    for codelist in prior_codelists.split("\n"):
+        if codelist:
+            requests_mock.get(
```

Comment:
Actually, I can comment out the call to requests_mock and the whole for loop and it still works (for all three tests). It's only the mock of the specific version that is needed. I assume this is a side-benefit of no longer relying on `update(..)`.

Reply:
Ah yes, I hadn't spotted that in the refactoring. You're entirely right, this call is no longer needed without the call to `update()` so I've removed it.